radix-engine-common/src/data/scrypto/custom_schema.rs

```diff
@@ -24,6 +24,8 @@ pub enum ScryptoCustomTypeKind {
     Decimal,
     PreciseDecimal,
     NonFungibleLocalId,
+
+    Reference,
```

Comment:
Can we rename this to `InternalRef` since `Address` is a reference in itself as well.

Reply:
Sure